docker/Dockerfile

```diff
@@ -58,13 +60,12 @@ ENV PATH="/opt/conda/envs/birdy/bin:$PATH"
 RUN python -m ipykernel install --name birdy && \
     (conda env export --name birdy 2>&1 | tee /conda-env-export-final.yml) && \
     (diff --unified /conda-env-export-phase1.yml /conda-env-export-final.yml 2>&1 | tee /conda-env-export.diff) && \
-    (du -sh /opt/conda 2>&1 | tee /conda-envs-size-final.txt)
+    (du -sh /opt/conda 2>&1 | tee /conda-envs-size-final.txt) && \
+    diff /conda-envs-size-phase1.txt /conda-envs-size-final.txt || true
 
-# install using same channel preferences as birdy original env to not downgrade
-# anything accidentally
-# this is for debug only, all dependencies should be specified in
-# environment.yml above
-# RUN mamba install -c conda-forge -c cdat -c bokeh -c plotly -c defaults -n birdy nbdime
+# install using same channel preferences as birdy original env to not downgrade anything accidentally
+# this is for debug only, all dependencies should be specified in environment.yml above
+# RUN mamba install -c conda-forge -c cdat -c bokeh -c plotly -n birdy nbdime
```

Comment:
Sorry just notice there is this list of channel to keep in-sync as well `-c conda-forge -c cdat -c bokeh -c fortiers`.  So 3 places to be in sync if you changes the list of channels.